craco.config.js

```diff
@@ -0,0 +1,29 @@
+const { ProvidePlugin } = require('webpack')
+
+module.exports = {
+  webpack: {
+    configure: {
+      resolve: {
+        fallback: {
+          path: require.resolve('path-browserify'),
+          timers: require.resolve('timers-browserify'),
+          fs: require.resolve('browserify-fs'),
+          https: require.resolve('https-browserify'),
+          stream: require.resolve('stream-browserify'),
+          http: require.resolve('stream-http'),
```

Comment:
Webpack 5 doesn't bundle these Node API polyfills that we need for various dependencies.